ports/fff/portfile.cmake

```
@@ -0,0 +1,12 @@
+# Header-only library
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO meekrosoft/fff
+    REF 7e09f07e5b262b1cc826189dc5057379e40ce886
```

```suggestion
    REF "v${VERSION}"
```
We can use the latest function.